### PR TITLE
Version 5.6.0 / Use jakarta.xml.bind instead of javax.xml.bind

### PR DESCRIPTION
**Pull request by jlgreve**

### CODE REVIEW

File: WHATSNEW.txt
Comment (Keymaster65) [inline WHATSNEW.txt:1]: The current solution is not "Breaking" or a "New feature" and the version then should be "5.5.1".

File: build.gradle
Comment (Keymaster65) [inline build.gradle:1]: I like the idea, to remove the dependency for "java.xml" 👍 

I would strongly recommend a solution using "java.util.Base64".

**Review by Keymaster65 — CHANGES_REQUESTED** [review]

File: build.gradle
Comment (jlgreve) [reply build.gradle:1]: I have replaced the implementation with java.util.Base64, marked the internal class as deprecated, replaced the internal usage of this class with direct usage of java.util.Base64, and removed the dependency to javax.xml.bind

File: WHATSNEW.txt [hunk 1]
```diff
@@ -1,3 +1,8 @@
+COPPER 5.5.1
+============
+- Deprecated: org.copperengine.core.util.Base64 (in favor of java.util.Base64)
+- Info      : Removed dependency on java.xml.bind in copper-coreengine
+
```
Comment (Keymaster65) [inline WHATSNEW.txt:5]: 👍

File: build.gradle [hunk 2]
```diff
@@ -426,9 +426,6 @@ project(':projects:copper-coreengine') {
         implementation "org.ow2.asm:asm-tree:$asmVersion"
         implementation "org.ow2.asm:asm-util:$asmVersion"
         implementation "org.ow2.asm:asm-analysis:$asmVersion"
-
-        implementation "javax.xml.bind:jaxb-api:$jaxbApiVersion"
-        implementation "com.sun.activation:javax.activation:$javaxActivationVersion"
     }
```
Comment (Keymaster65) [inline build.gradle:429]: 👍

File: gradle.properties [hunk 3]
```diff
@@ -25,7 +24,6 @@ mockitoVersion = 5.11.0
 byteBuddyVersion = 1.14.13
 hamcrestVersion = 2.2
 
-jaxbApiVersion = 2.3.1
 jgitVersion = 6.9.0.202403050737-r
```
Comment (Keymaster65) [inline gradle.properties:27]: 👍

File: projects/copper-coreengine/src/main/java/org/copperengine/core/persistent/StandardJavaSerializer.java [hunk 4]
```diff
@@ -37,9 +37,10 @@
  * @author austermann
  */
 public class StandardJavaSerializer implements Serializer {
-
     public static final boolean DEFAULT_COMPRESS = true;
 
+    private static final Base64.Encoder B64_ENCODER = Base64.getEncoder();
+    private static final Base64.Decoder B64_DECODER = Base64.getDecoder();
     private static final String COPPER_3_PACKAGE_PREFIX = "org.copperengine.core.";
```
Comment (Keymaster65) [inline projects/copper-coreengine/src/main/java/org/copperengine/core/persistent/StandardJavaSerializer.java:44]: Gefällt mir sehr gut 👍

File: projects/copper-coreengine/src/main/java/org/copperengine/core/util/Base64.java [hunk 5]
```diff
@@ -15,25 +15,29 @@
...
-
 /**
  * Provides Base64 encoding and decoding as defined by RFC 2045.
  *
  * @author dmoebius
  * @since 3.0.1
+ * @deprecated This class can be replaced by using the built-in class java.util.Base64
  */
+@Deprecated
 public class Base64 {
```
Comment (Keymaster65) [inline projects/copper-coreengine/src/main/java/org/copperengine/core/util/Base64.java:26]: Perfect 👍

**Review by Keymaster65 — APPROVED** [review]

**Pull request merged**